test-packages/test-utils/api.d.ts

```diff
@@ -119,3 +124,17 @@ export function expectSuggestions(
   suggestions: UI5XMLViewCompletion[],
   expected: string[]
 ): void;
+
+export function readTestLibraryFile(
```

Comment:
we should consider (in future) having multiple `test-packages` this one is becoming a bit of a monolith

Reply:
We should probably at least separate the model build utilities and the model generation utilities